.github/workflows/pull_request.yml

```diff
@@ -17,10 +17,12 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Install Testing Requirements
-        run: sudo pip install -r requirements-test.txt
+        run: |
+          sudo pip install -r requirements-test.txt
+          sudo curl -sL -o /bin/hadolint "https://github.com/hadolint/hadolint/releases/download/v1.16.3/hadolint-Linux-x86_64" && sudo chmod +x /bin/hadolint
```

Comment:
\imo: The two commands can be in different lines for readability.

Reply:
I didn't separated the two command because it is used for installing one dependency `hadolint`. I will change this for readability purpose.